release-notes/unreleased/LUN-440-select-image-from-gallery.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-440
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Nutzer:innen können nun Bilder für eigene Vokablen aus der Fotomediathek benutzen.
```

Comment:
Please delete these release notes, there will be one release note for the whole own vocabulary feature.